scarb-metadata/src/lib.rs

```diff
@@ -534,6 +534,21 @@ impl Metadata {
         self.packages.iter().find(|p| p.id == *id)
     }
 
+    /// Checks if the plugin associated with [`CompilationUnitCairoPluginMetadata`] is built-in into the compiler.
+    pub fn is_builtin(&self, plugin: &CompilationUnitCairoPluginMetadata) -> Option<bool> {
```

Comment:
```suggestion
    pub fn is_builtin_plugin(&self, plugin: &CompilationUnitCairoPluginMetadata) -> Option<bool> {
```
nit: `Metadata::is_builtin` would suggest to me, that metadata itself can be builtin or not